pkg/test/vpc_client/proxy.go

```diff
@@ -46,20 +47,39 @@ func (vpc *VPC) LaunchProxyInstance(imageID string, zone string, sshKey string)
 		return inst, "", "", err
 	}
 
-	instOut, err := vpc.AWSClient.LaunchInstance(pubSubnet.ID, imageID, 1, "t3.medium", CON.InstanceKeyName, []string{SGID}, true)
+	keyName := fmt.Sprintf("%s-%s", CON.InstanceKeyNamePrefix, keypairName)
+	key, err := vpc.CreateKeyPair(keyName)
 	if err != nil {
-		log.LogError("Launch proxy instance failed %s", err)
+		log.LogError("Create key pair %s failed %s", keyName, err)
 		return inst, "", "", err
-	} else {
-		log.LogInfo("Launch proxy instance %s succeed", *instOut.Instances[0].InstanceId)
 	}
 	tags := map[string]string{
 		"Name": CON.ProxyName,
 	}
+	_, err = vpc.AWSClient.TagResource(*key.KeyPairId, tags)
+	if err != nil {
+		log.LogError("Add tag for key pair %s failed %s", *key.KeyPairId, err)
+		return inst, "", "", err
+	}
+	sshKey, err := file.WriteToFile(*key.KeyMaterial, "proxyKeyPair.pem", privateKeyPath)
```

Comment:
Let's not hardcode the keyName

Reply:
Fixed it. Would you help review it again?